javascript/step_1.html

```diff
@@ -1,14 +1,17 @@
 <div class="instructions">
-    <p>Welcome to Pokerwars! In this tutorial you will learn how to build a simple bot. You can switch your programming language anytime with the dropdown selector at the top right. Let's get started!</p>
-    <p>Remember, <a href="https://github.com/pokerwars" target="_blank">starter bots</a> are always available if you prefer to run your bot locally.</p>
-    <h2>Setting up your environment</h3>
-    <h3>The starter projects</h3>
-    <p>In our GitHub repository you can find starter projects for a number of languages. You are not limited to those,
-        you can use any programming language you want as long as it's capable of sending and receiving HTTP requests.
-    </p>
-    <p>For Javascript, our starter project is <a href="https://github.com/pokerwars/pokerwars.io-starterbot-javascript"
-            target="_blank">here</a>. We will build that same starter bot now step by step. At the end, you will get
-        an understanding of the game and the platform, and a working bot ready for you to implement your own unique
-        strategy. How cool is that!
-    </p>
+  <p>Welcome to pokerwars! In this tutorial you will learn how to build a simple bot. Let's get
+    started!</p>
+  <h2>The starter bots</h2>
+  <p>In our <a href="https://github.com/pokerwars" target="_blank">GitHub page</a> you can find
+    starter bots for a number of languages. These are very simple versions of pre-built bots. You
```

Comment:
what about changing `These are very simple versions of pre-built bots` into `These are very simple pre-build bots`